src/models/invitation.model.ts

```diff
@@ -27,14 +27,14 @@ const InvitationSchema = new Schema({
     {
       email: {
         type: String,
-        requried: false,
+        required: true,
```

Comment:
Why are adding  required

Reply:
Email is needed in all invitation processing Sir